### DISCUSSION TITLE
Doughnut - Is it possible to have the label curve with the arc.

### DISCUSSION THREAD

**Discussion by scurenton** [chartjs/chartjs-plugin-datalabels#365] [Q&A]
I know how to rotate the label and all that, but the label is just flat on a straight line.  Does anyone know if it is possible to have the label curve with the arc. like the words Planning or Recruitment... in the image here: 
![Capture](https://user-images.githubusercontent.com/16226502/212204682-972a0948-6e8d-4311-8acd-69583bc17e03.PNG)

**Comment by Udemology** [comment]
I have created an option but it is not with the datalabels plugin. I could not figure out how to add curved text so I made a custom solution: https://youtu.be/SzgPFDGSRKY

See if that would help.
